crates/turborepo-ui/src/tui/task.rs

```diff
@@ -70,9 +70,17 @@ impl Task<Running> {
 }
 
 impl Task<Finished> {
-    pub fn start(&self) -> Instant {
+    pub fn start_time(&self) -> Instant {
         self.state.start
     }
+    pub fn start(self) -> Task<Running> {
```

Comment:
This feels off, do we want to start running tasks again vs creating a new task and having it go through the natural progression of `planned -> running -> finished`?

Reply:
Ahh yeah that makes more sense